event_manager/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.contrib.auth.models import User
+
+class member(models.Model):
+    user=models.ForeignKey(User,on_delete=models.CASCADE)
+    last_name=models.CharField(max_length=200)
```

Comment:
Why do we only have "last name" here? Shouldn't we have first and last name?

Reply:
first name is in the auth user already, i dont think we need to duplicate that

Reply:
Ah, I see. Can we also remove last name then. I believe the Django user class includes lastname. Nice thinking trying to find which fields are unnecessary.

Reply:
i didn't see the lastname in the user class, can you check again and if its there, we can remove it

Reply:
Looks like User class contains lastname: https://docs.djangoproject.com/en/2.0/ref/contrib/auth/.

Reply:
alright we can take the last name out then